libs/src/api/serviceProvider.js

```diff
@@ -105,8 +105,16 @@ class ServiceProvider extends Base {
 
     let services = {}
     timings.forEach(timing => {
-      services[timing.request.id] = timing.response.data
+      if (timing.response) {
+        services[timing.request.id] = timing.response.data
+      } else {
+        services[timing.request.id] = undefined
```

Comment:
what happens if it's unhealthy and we add this entry to the map? wanna double check that this is a safer assumption to make versus just not adding it to `services` at all

Reply:
discussed offline -- will break the UI :-). updating code to not add services with undefined as the response